### ISSUE TITLE
JS Problem?

### ISSUE THREAD

**Issue by foomep** [janizde/WP-Opening-Hours#43]
Since the update to Wordpress 4.7 (I think) I have no datepicker and can't add or delete Holidays or Irregular Opening. I think I had this problem with a Beta version.
It's a Javascript Bug?

**Comment by misagues** [comment]
I have tested with the version of WordPress 4.7 and the datepicker works
![datepicker_wp47](https://cloud.githubusercontent.com/assets/16360881/21459904/0d75b8fc-c943-11e6-8459-375f8fb8b511.JPG)

**Comment by janizde** [comment]
Are there any JS errors in your browser's console?

**Comment by foomep** [comment]
ahh ok. I have some problems in the console:
```
Laden von gemischten aktiven Inhalten "http://code.jquery.com/ui/1.11.4/jquery-ui.min.js?ver=4.7" wurde blockiert.[Weitere Informationen] edit.php

JQMIGRATE: Migrate is installed, version 1.4.1  load-scripts.php:9:542

TypeError: t.datepicker is not a function[Weitere Informationen]  main.js:1:27000
```

I have change from http to https a while ago. I think thats the problem. sry

**Comment by janizde** [comment]
fixed in 2.0.1
thank you
